models/order.js

```diff
@@ -12,7 +13,68 @@ const Entry = new Schema({
 const orderSchema = new Schema({
   id: String,
   status: { type: String, default: constants.OPEN },
-  items: [Entry]
+  entries: [Entry]
 });
 
+orderSchema.statics.getOrders = function() {
+  return getOrders.apply(this)
```

Comment:
Ah, I guess `this` is referring to `orderSchema` huh..? Looks like we need apply then.